draft-vvv-webtransport-quic.md

```diff
@@ -129,23 +131,70 @@ the client receives a TLS Finished message from the server.
 
 In order to identify itself as a WebTransport application, QuicTransport relies
 on TLS Application-Layer Protocol Negotiation {{!RFC7301}}.  The user agent MUST
-request the ALPN value of "wq" and it MUST NOT establish the session unless that
-value is accepted.
+request the ALPN value of "wq-draft01" and it MUST NOT establish the session
```

Comment:
I'd personally prefer `wq-01` to match the QUIC docs

Reply:
I actually changed it to `wq-vvv-01`, since this is not a WG draft.

Reply:
SGTM